docs/MODES.md

```diff
@@ -48,6 +48,14 @@ LotsOfPointsMode.onKeyUp = function(state, e) {
 LotsOfPointsMode.toDisplayFeatures = function(state, geojson, display) {
   display(geojson);
 };
+
+// Add the new draw mode to the mapbox Draw object
```

Comment:
This would setup draw without any other modes and `lots_of_points` depends on `simple_select` which depends on `direct_select` so we should include those as well.